docs/source/status.rst

```diff
@@ -0,0 +1,199 @@
+.. KIND, either express or implied.  See the License for the
+.. specific language governing permissions and limitations
+.. under the License.
+
+=====================
+Implementation Status
+=====================
+
+The following tables summarize the features available in the various official
+Arrow libraries.  Unless otherwise stated, the Python, R, Ruby and C/GLib
+libraries follow the C++ Arrow library.
+
+Data Types
+==========
+
++-------------------+-------+-------+-------+------------+-------+-------+
+| Data type         | C++   | Java  | Go    | JavaScript | C#    | Rust  |
+| (primitive)       |       |       |       |            |       |       |
++===================+=======+=======+=======+============+=======+=======+
+| Null              | ✓     | ✓     |       |            |       | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Boolean           | ✓     | ✓     | ✓     | ✓          |  ✓    | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Int8/16/32/64     | ✓     | ✓     | ✓     | ✓          |  ✓    | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| UInt8/16/32/64    | ✓     | ✓     | ✓     | ✓          |  ✓    | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Float16           |       |       | ✓     |            |       |       |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Float32/64        | ✓     | ✓     | ✓     | ✓          |  ✓    | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Decimal128        | ✓     | ✓     |       |            |       |       |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Date32/64         | ✓     | ✓     | ✓     | ✓          |  ✓    | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Time32/64         | ✓     | ✓     | ✓     | ✓          |       | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Timestamp         | ✓     | ✓     | ✓     | ✓          |  ✓    | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Duration          | ✓     | ✓     | ✓     |            |       | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Interval          | ✓     | ✓     | ✓     |            |       | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Fixed Size Binary | ✓     | ✓     | ✓     | ✓          |       | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Binary            | ✓     | ✓     | ✓     | ✓          |  ✓    | ✓     |
++-------------------+-------+-------+-------+------------+-------+-------+
+| Large Binary      | ✓     | ✓     | ✓     | ✓          |       |       |
```

Comment:
R doesn't have bindings for the Large* types yet (ARROW-6543)

Reply:
I don't know where to add that information. What would you suggest? Add columns for R and Python (but then we also need Ruby and C/GLib...)?

Reply:
🤷 it says they follow C++ "unless otherwise stated" so I figure that means another `(4)` like footnote.

Also I don't know .rst well, can you add comments? Might be worthwhile to note (internally, not publicly) the JIRAs that will resolve exceptions like these so that we can more easily in the future know if we can remove the notes.

Reply:
You can add comments, yes.
https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#comments